wordpress-mobile/WordPress-iOS
# UL&S: Enable the unified Site Address feature flag for local dev builds

Fixes #14170

This PR adds a new feature flag to be used with Unified Auth. It's currently only enabled for `.localDeveloper` builds.

### To test
Nothing is changing in the UI or navigation at this time.

If you'd like to verify nothing has changed:
1. Log out if logged in
2. Log in > "Or log in by entering your site address."
**Expected:** this still navigates to the old flow.


PR submission checklist:
